enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -5181,6 +5181,8 @@ Value *GradientUtils::invertPointerM(Value *const oval, IRBuilder<> &BuilderM,
                 chunk = 8;
               } else if (flt->isHalfTy()) {
                 chunk = 2;
+              } else if (flt->isX86_FP80Ty()) {
```

Comment:
and here

Reply:
also changed in [24c8098](https://github.com/EnzymeAD/Enzyme/pull/1604/commits/24c8098f99d83af45a0d681a44d9ec9411d40be3)